opencti-platform/opencti-front/src/components/fields/SelectField.jsx

```diff
@@ -80,6 +80,7 @@ const SelectField = (props) => {
         style={{ color: props.disabled ? '#4f4f4f' : '' }}
         variant={props.variant}
         id={labelId}
+        required
```

Comment:
should it come from a prop `required` that defaults to false, like other fields ?